.travis_scripts/build_all

```diff
@@ -1,13 +1,14 @@
 #!/usr/bin/env bash
 
+set -ex
 
 # Fast finish the PR.
 echo ""
 (curl https://raw.githubusercontent.com/conda-forge/conda-forge-ci-setup-feedstock/master/recipe/ff_ci_pr_build.py | \
     python - -v --ci "travis" "${TRAVIS_REPO_SLUG}" "${TRAVIS_BUILD_NUMBER}" "${TRAVIS_PULL_REQUEST}") || exit 1
 
 # Set the numpy variable. This isn't used, but conda-build complains if we haven't set it already.
-export CONDA_NPY=19
+export CONDA_NPY=111
```

Comment:
I think this hack is no longer needed. It predates conda-build 3. Would submit a separate PR to just remove it.

Reply:
Thanks: https://github.com/conda-forge/staged-recipes/pull/7427